manifest.json

```diff
@@ -11,8 +11,12 @@
         "default_popup": "popup/popup.html",
         "default_icon": "resources/vibinex-logo.png"
     },
+    "host_permissions": [
+        "*://*/*"
+    ],
```

Comment:
Needed for cookies and cors to work. Tried a bunch of narrower permissions but didnt work. Leaving it like this for now

Reply:
Please really try. This is a serious permission. Chrome Web Store will block our publish to review this and they will ask for a reason to allow host-permissions for all URLs. This is also shown to any new users who are installing the extension. And asking for host permissions for all possible URLs is not going to look good and we will see higher drop-offs than necessary. 

LOGAF: 10/10

Reply:
Took care of it